kolibri/core/hooks.py

```diff
@@ -0,0 +1,40 @@
+"""
+Kolibri Core hooks
+------------------
+
+WIP! Many applications are supposed to live inside the core namespace to make
+it explicit that they are part of the core.
+
+Do we put all their hooks in one module or should each app have its own hooks
+module?
```

Comment:
We had bandied around the idea of auto-detecting core-plugins using a "KNOWN_PLUGINS" idea or something similar. Is it time to implement that?